.github/workflows/ci_workflow.yml

```diff
@@ -329,7 +329,7 @@ jobs:
             build-shared: 'ON'
             compiler-desc: AppleClang11.0
             cxx-standard: 17
-            osver: 11.0
```

Comment:
But isn't CY2023 macOS 11? I thought what we needed to do was to remove the CY2022 job altogether, since it specifies 10.5 which GHA no longer supports?  And maybe add a single macOS 12 job, since it's newer?  If not, can you fix the comment to match the code?

Reply:
You are right - I forgot they specify that now in the platform base, was just going off I had a bunch of jobs timing out trying to get to a macos 10.15 host. I've updated / fixed